### PR TITLE
one more test

### PR DESCRIPTION
**Pull request by lglof**
just added a test for get entries route, ensures error when requested id doesn't have any associated interventions.

### CODE REVIEW

File: src/routes.py [hunk 1]
```diff
@@ -22,11 +22,14 @@ def createEntry():
 @cross_origin()
 def getEntries(num, id):
     interventions = interventionFunctions.viewInterventions(num, id)
-    data = {
-        'items': [item.to_dict() for item in interventions],
-        'num_items': len(interventions)
-    }
-    response = make_response(data, 200)
+    if (interventions == 'error'):
+        response = make_response({"error_message": "id does not exist"}, 400)
```
Comment (asilverfox) [inline src/routes.py:26]: Is this supposed to be a 404? (Basing off of the test that expects a 404)

File: src/interventionFunctions.py [hunk 2]
```diff
@@ -21,7 +21,10 @@ def addIntervention(contents):
 
 def viewInterventions(num, idNo):
     interventions = PerformedIntervention.query.filter_by(patient_id=idNo).order_by(PerformedIntervention.time.desc()).limit(num).all()
-    return interventions
+    if (len(interventions) == 0):
+        return 'error'
```
Comment (asilverfox) [inline src/interventionFunctions.py:25]: Would this throw an error for any new patient?

**Review by asilverswan — CHANGES_REQUESTED** [review]
Small little issues that should probably be addressed.

File: src/routes.py (hunk 1 above)
Comment (lglof) [reply src/routes.py:26]: whoops, I updated this in the test file and then forgot to save it so git didn't recognise that it had been changed.

File: src/interventionFunctions.py (hunk 2 above)
Comment (lglof) [reply src/interventionFunctions.py:25]: Hmm, yeah
What if updated the error message to say something more like 'there are no interventions recorded for this patient'

Comment (asilverfox) [reply src/interventionFunctions.py:25]: Yeah, I'm good with that. The only other thing is to check if patient id exists if you fail to retrieve any interventions.

**Review by asilverswan — APPROVED** [review]
lgtm

**Pull request merged**